app/src/main/java/com/example/erabook/util/Extensions.kt

```diff
@@ -12,4 +14,8 @@ fun ImageView.loadImage(imageLink: String?) {
             .load(absolutePath)
             .into(this)
     }
+}
+
+fun Context.showToast(messageResId: Int, duration: Int = Toast.LENGTH_SHORT) {
```

Comment:
Please annotate messageResId with @StringRes in order to accept only string resources and not normal Int values.